### ISSUE TITLE
Failure to open on Fedora 38

### ISSUE THREAD

**Issue by trprado** [flathub/com.microsoft.EdgeDev#5]
I have tried to reinstall, but the error still appears.This error only occurs in x11. It works fine in wayland.

OS: Fedora 38
Flatpak Version: 1.15.4

```bash
$ flatpak run com.microsoft.EdgeDev
Gtk-Message: 08:33:30.570: Failed to load module "canberra-gtk-module"
Gtk-Message: 08:33:30.570: Failed to load module "pk-gtk-module"
Gtk-Message: 08:33:30.570: Failed to load module "canberra-gtk-module"
Gtk-Message: 08:33:30.570: Failed to load module "pk-gtk-module"

** (flextop-init:9): WARNING **: 08:33:30.576: Failed to check desktop file: Key file contains line ?\u000bvia-trprado??????????????????????????????????????????\u000b?v?i?a?-?t?r?p?r?a?d?o?????????????????????????????????????????????????????????????????????????????????????? which is not a key-value pair, group, or comment
Stub sandbox ignoring command: /app/extra/nacl_helper
[2:2:0502/083330.723442:ERROR:nacl_fork_delegate_linux.cc(313)] Bad NaCl helper startup ack (0 bytes)
[0502/083330.730534:ERROR:scoped_ptrace_attach.cc(27)] ptrace: Operation not permitted (1)
```

**Comment by eyenpi** [comment]
I couldn't reproduce it on Archlinux. It seems the problem exists in the stable version too but not in the beta. So maybe it's an upstream problem. Check the link below and see if any of the solutions work for you.
https://github.com/flathub/com.microsoft.Edge/issues/304

**Comment by GreenCappuccino** [comment]
This issue seems to have been resolved in the latest release of Edge (dev): 114.0.1823.4-1

**Comment by eyenpi** [comment]
It seems the problem is fixed upstream.
